helm/charts/nats/Chart.yaml

```
@@ -6,7 +6,7 @@ keywords:
 - nats
 - messaging
 - cncf
-version: 1.1.3
+version: 1.2.0
```

WIll put in follow-up PR that adds a test and bumps

```suggestion
version: 1.1.3
```